README.md

```diff
@@ -652,6 +653,51 @@ The supported Picture in Picture events on `PlayerView` are:
 
 Check [`events.ts`](https://github.com/bitmovin/bitmovin-player-react-native/blob/development/src/components/PlayerView/events.ts) for more information about them.
 
+### Customize HTML UI (Android and iOS only)
+
+The Bitmovin Player SDKs use the open source [Bitmovin Player Web UI](https://github.com/bitmovin/bitmovin-player-ui) on all platforms, except tvOS.
+This is cusomizable in multiple ways.
+
+#### Custom implementation
+
+Since the Bitmovin Player Web UI is open source, it can be forked and modified to tailor to any application's needs.
+See [Cusomizing the UI](https://github.com/bitmovin/bitmovin-player-ui#customizing-the-ui) section for details.
+
+In case a custom implementation of the Player UI is desired, configure the hosted JS and CSS files via the `StyleConfig`, like:
```

Comment:
Nit wording
```suggestion
In case a custom implementation of the Player UI is desired, configure the hosted JS and CSS files via the `StyleConfig` as shown in the following example:
```